views/cart.php

```diff
@@ -0,0 +1,116 @@
+<div class="cart__container">
+    <h4>Your Cart</h4>
+    <div class="cart__table">
+        <table id="cartTable">
+            <tr>
+                <th></th>
+                <th>Product</th>
+                <th>Price</th>
+                <th>Quantity</th>
+                <th>Total</th>
+            </tr>
+
+            <tr>
+                <td class="cart__item__remove">
+                    <i class="far fa-trash-alt"></i>
+                </td>
+                <td>
+                    <img src="./assets/Images/background.jpg" alt="">
+                    Product 1
+                </td>
+                <td>000.000.000</td>
+                <td class="cart__item__quantity">
+                    <input type="number" value="1" step="1" min="1" max="100" class="input-group">
+                </td>
+                <td class="cart__item__total">000.000.000</td>
+            </tr>
+            <tr>
+                <td class="cart__item__remove">
+                    <i class="far fa-trash-alt"></i>
+                </td>
+                <td>
+                    <img src="./assets/Images/background.jpg" alt="">
+                    Product 1
+                </td>
+                <td>000.000.000</td>
+                <td class="cart__item__quantity">
+                    <input type="number" value="1" step="1" min="1" max="100" class="input-group">
+                </td>
+                <td class="cart__item__total">000.000.000</td>
+            </tr>
+            <tr>
+                <td class="cart__item__remove">
+                    <i class="far fa-trash-alt"></i>
+                </td>
+                <td class="cart__item__name">
+                    <img src="./assets/Images/background.jpg" alt="">
+                    Product 1
+                </td>
+                <td>000.000.000</td>
+                <td class="cart__item__quantity">
+                    <input type="number" value="1" step="1" min="1" max="100" class="input-group">
+                </td>
+                <td class="cart__item__total">000.000.000</td>
+            </tr>
+        </table>
+    </div>
+</div>
+<script>
+    $(function() {
+        $.get("/cartItems", function(data) {
+            console.log(typeof(data));
+            var itemsArray = JSON.parse(data);
+            $.each(itemsArray, function(key, value) {
+                var cart_item_id = value.id;
+                var product_id = value.product_id;
+                var product_quantity = value.quantity;
+
+                $.get(`/productInfo?id=${product_id}`, function(data) {
```

Comment:
Since you are relying on ES6 for string interpolation, you may as well use the destructuring assignment for the above:

```js
var { id: cart_item_id, product_id, quantity: product_quantity } = value;
```